test/torchaudio_unittest/prototype/functional/autograd_test_impl.py

```diff
@@ -28,3 +28,53 @@ def test_add_noise(self):
 
         self.assertTrue(gradcheck(F.add_noise, (waveform, noise, lengths, snr)))
         self.assertTrue(gradgradcheck(F.add_noise, (waveform, noise, lengths, snr)))
+
+
+class AutogradTestRayTracingImpl(TestBaseMixin):
```

Comment:
nit: we can add the tests under the same `AutogradTestImpl` since they all belong to functional category.

Reply:
Sounds good - should I do the same for `FunctionalTestRayTracingImpl` and move it under `FunctionalTestImpl` ?

Reply:
sure.